modules/library/main/src/main/java/org/geotools/util/CssColorConverter.java

```diff
@@ -0,0 +1,202 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ * 
+ *    (C) 2017, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.util;
+
+import java.awt.Color;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.geotools.factory.Hints;
+import org.geotools.util.Converter;
+import org.geotools.util.ConverterFactory;
+
+/**
+ * 
+ * Converts CSS Color Module 4 names to {@link Color} objects.
+ * 
+ * @see <a href="https://www.w3.org/TR/css-color-4/#named-colors">https://www.w3.org/TR/css-color-4/#named-colors</a> 
+ *
+ */
+public class CssColorConverter implements Converter {
+
+    private static final Map<String, Color> CSS_COLORS;
```

Comment:
This should be a public field on ColorConverter so that the gt-ysld and gt-css modules can make use of this map (rather than have their own duplicate).